bundles/org.openhab.binding.network/.classpath

```diff
@@ -28,5 +28,27 @@
 			<attribute name="maven.pomderived" value="true"/>
 		</attributes>
 	</classpathentry>
+	<classpathentry kind="src" path="target/generated-sources/annotations">
```

Comment:
why are these classoath-entries added?

Reply:
good question ....?...

Reply:
I've used VSCode to make these changes, I suspect it comes from it. Let me correct that.